extensions/amp-story-share-menu/0.1/amp-story-share-menu.js

```diff
@@ -360,3 +359,17 @@ export class AmpStoryShareMenu {
     );
   }
 }
+
+/**
+ * This extension installs the share widget.
+ */
+
+AMP.extension('amp-story-share-menu', '0.1', (AMP) => {
+  AMP.ampdoc.whenReady().then(() => {
+    const element = AMP.ampdoc
+      .getRootNode()
+      .querySelector(`.${escapeCssSelectorIdent(SHARE_MENU_HOST_CLASS)}`);
+    const shareMenu = new AmpStoryShareMenu(element, AMP.ampdoc.win);
```

Comment:
Plz add a comment explaining that, one day, when we have 100 engineers and unlimited cycles, we should migrate this to being an AMP component